express/scripts/gnav.js

```diff
@@ -69,7 +69,12 @@ function loadIMS() {
     locale: getLocale(window.location),
     environment: 'prod',
   };
-  loadScript('https://auth.services.adobe.com/imslib/imslib.min.js');
+  if (!['www.stage.adobe.com'].includes(window.location.hostname)) {
+    loadScript('https://auth.services.adobe.com/imslib/imslib.min.js');
+  } else {
+    loadScript('https://auth.services.adobe.com/imslib/imslib.min.js');
+    window.adobeid.environment = 'stg1';
```

Comment:
if the loadScript is extracted, we could also reduce this into a ternary.

P.S. Not sure what is in imslib.min.js but if the script has self-invoking calls and it uses values from the window.adobeid object, should we be configuring the adobeid.environment before loading the script?

Reply:
https://github.com/adobecom/milo/blob/1eb0bcf99bb58d2fc0488ddd1077ffa3f18c4cf5/tools/utils/utils.js#L4-L14 that is how they invoke it using await
Probably we should do the same first by setting the values..
wdyt?

Reply:
Yeah it seem script would try to read from `window.adobeid`. We would be safe to configure it first before loading the script.